### ISSUE TITLE
Banners on site are not accessible

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#12758]
**Describe the bug**

I cam across this while working on #11983. The banner to finish setting up your environment is not accessible. The banner should have a `role` attribute with the value of "alert". See [Using the alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role). In fact all the banners on the site should have it.

The attached screenshot is complaining the banner is not in the main landmak, but that's a false positive, the real issue is the alert role.

**To Reproduce**

**Expected behavior**

Banners on our site should show up as an alert when using voice over and show up in the accessibility tree as an alert.

**Screenshots**

<img width="2039" alt="Screen Shot 2021-02-21 at 7 41 44 PM" src="https://user-images.githubusercontent.com/833231/108644427-e44dc200-747c-11eb-8db1-a963f5427247.png">

**Desktop (please complete the following information):**

- OS, version: N/A
- Browser, version: N/A

**Smartphone (please complete the following information):**

- Device: N/A
- OS, version: N/A
- Browser, version: N/A

**Additional context**

**Comment by nickytonline** [comment]
One thing I'm wondering @aitchiss is if it makes more sense to use `aria-live="polite"` as not all banners are urgent. Thoughts?

**Comment by aitchiss** [comment]
> One thing I'm wondering @aitchiss is if it makes more sense to use `aria-live="polite"` as not all banners are urgent. Thoughts?

Yep I think `aria-live="polite"` in this scenario would be best, as it's not totally critical a user's alerted to it immediately. I think like you say there are a lot of banners/notices we use that could benefit from this 👍
